pkg/apis/certmanager/validation/issuer.go

```diff
@@ -299,6 +299,24 @@ func ValidateACMEIssuerDNS01Config(iss *v1alpha1.ACMEIssuerDNS01Config, fldPath
 				}
 			}
 		}
+		if p.Webhook != nil {
+			if numProviders > 0 {
+				el = append(el, field.Forbidden(fldPath.Child("webhook"), "may not specify more than one provider type"))
+			} else {
+				numProviders++
+				fldPath = fldPath.Child("webhook")
+
+				if len(p.Webhook.URL) == 0 {
+					el = append(el, field.Required(fldPath.Child("url"), ""))
+				} else if _, err := url.ParseRequestURI(p.Webhook.URL); err != nil {
+					el = append(el, field.Invalid(fldPath.Child("url"), "", fmt.Sprintf("\"url\" field does not contain correct URL")))
```

Comment:
`\"url\" field does not contain correct URL` -> `\"url\" field does not contain a valid URL: %v` (and include the error message 😄)